containers/foreman.yml

```diff
@@ -8,6 +8,12 @@
     pulp_worker_count: 2
     registry: quay.io/foreman
     deployment_state: present
+    enabled_services:
+      - postgres
+      - puppet
+      - pulp
+      - candlepin
+      - qpid
```

Comment:
Can actually drop qpid here since Candlepin and Pulp require it.

Reply:
should I remove it from foreman/defaults/main.yml as well then? Without qpid in this list it won't be included by foreman.